### PR TITLE
Omemo signal protocol

### PR DESCRIPTION
**Pull request by strb**
These are the same changes @iNPUTmice proposed in #470, with a small change in the wording.

### CODE REVIEW

File: xep-0384.xml [hunk 1]
```diff
@@ -87,14 +100,14 @@
...
-      included once, reducing overhead. Furthermore, Olm's transparent handling
+      included once, reducing overhead. Furthermore, SignalProtocols’s transparent handling
       of messages that were lost or received out of order, as well as those sent
       while the recipient was offline, is maintained by this protocol. As a
       result, in combination with &xep0280; and &xep0313;, the desired property
       of inter-client history synchronization is achieved.
     </p>
     <p>
-      OMEMO currently uses version 1 Olm protocol. Instead of an Axolotl key
+      OMEMO currently uses version 3 SignalProtocol. Instead of an Signal key
```
Comment (jcbrand) [inline xep-0384.xml]: Typo, should be "a".

File: xep-0384.xml [hunk 2]
```diff
@@ -216,10 +229,10 @@
     to='juliet@capulet.lit'
     id='fetch1'>
   <pubsub xmlns='http://jabber.org/protocol/pubsub'>
-    <items node='urn:xmpp:omemo:0:bundles:31415'/>
+    <items node='eu.siacs.conversations.axolotl.bundles:31415'/>
   </pubsub>
 </iq>]]></example>
-    <p>A random preKeyPublic entry is selected, and used to build an Olm session.</p>
+    <p>A random preKeyPublic entry is selected, and used to build an SignalProtocol session.</p>
```
Comment (jcbrand) [inline xep-0384.xml]: Typo, should be "a".

**Comment by horazont** [comment]
@strb Please [sign the CLA using the CLAbot](https://cla-assistant.io/xsf/xeps?pullRequest=482).

**Comment by Sur3** [comment]
How did this get through, instead of https://github.com/xsf/xeps/pull/463.
Why was olm/matrix removed again now? o.O

**Comment by horazont** [comment]
@Sur3 Please move technical discussions to the [standards mailing list](https://mail.jabber.org/pipermail/standards/).

**Pull request merged**